Package.swift

```diff
@@ -280,5 +280,9 @@ let package = Package(
             name: "SmithyRetriesTests",
             dependencies: ["ClientRuntime", "SmithyRetriesAPI", "SmithyRetries", "SmithyTestUtil"]
         ),
+        .testTarget(
+            name: "SmithyHTTPAPITests",
+            dependencies: ["SmithyHTTPAPI"]
+        ),
```

Comment:
Added a new test target & moved the `Headers` tests into it, since `Headers` is in the `SmithyHTTPAPI` module.